### PR TITLE
Move the reindex function to dev-tools helper class

### PR DESCRIPTION
**Pull request by shadyvb**
Requires https://github.com/humanmade/altis-dev-tools/pull/311
Ref https://github.com/humanmade/altis-cms/issues/508

### CODE REVIEW

File: tests/acceptance/ElasticSearchCest.php [hunk 1]
```diff
@@ -101,7 +101,7 @@ public function testSimpleModeSynonymSearch( AcceptanceTester $I ) {
 			'synonyms-text' => 'cat, dog, hamster',
 		] );
 
-		$this->_index();
+		$I->reindexElastic();
```
Comment (shadyvb) [inline tests/acceptance/ElasticSearchCest.php:104]:
```suggestion
		$I->reindexContent();
```

File: tests/acceptance/ElasticSearchCest.php [hunk 2]
```diff
@@ -141,7 +141,7 @@ public function testAdvancedModeSynonymSearch( AcceptanceTester $I ) {
 			'synonyms-text' => "cat, dog, hamster\nsneaker, trainer, shoe, loafer",
 		] );
 
-		$this->_index();
+		$I->reindexElastic();
```
Comment (shadyvb) [inline tests/acceptance/ElasticSearchCest.php:144]:
```suggestion
		$I->reindexContent();
```

File: tests/acceptance/ElasticSearchCest.php [hunk 3]
```diff
@@ -180,7 +180,7 @@ public function testStopwords( AcceptanceTester $I ) {
 			'stopwords-text' => 'ignore',
 		] );
 
-		$this->_index();
+		$I->reindexElastic();
```
Comment (shadyvb) [inline tests/acceptance/ElasticSearchCest.php:183]:
```suggestion
		$I->reindexContent();
```

File: tests/acceptance/ElasticSearchCest.php [hunk 4]
```diff
@@ -206,7 +206,7 @@ public function testUserSearch( AcceptanceTester $I ) {
 			],
 		] );
 
-		$this->_index();
+		$I->reindexElastic();
```
Comment (shadyvb) [inline tests/acceptance/ElasticSearchCest.php:209]:
```suggestion
		$I->reindexContent();
```

File: tests/acceptance/ElasticSearchCest.php [hunk 5]
```diff
@@ -255,7 +255,7 @@ public function testCustomSearchResults( AcceptanceTester $I ) {
 			] );
 		}
 
-		$this->_index();
+		$I->reindexElastic();
```
Comment (shadyvb) [inline tests/acceptance/ElasticSearchCest.php:258]:
```suggestion
		$I->reindexContent();
```

File: tests/acceptance/ElasticSearchCest.php [hunk 6]
```diff
@@ -295,7 +295,7 @@ public function testReusableBlockContent( AcceptanceTester $I ) {
 			'post_content' => sprintf( '<!-- wp:block {"ref":%d} /-->', $block_id ),
 		] );
 
-		$this->_index();
+		$I->reindexElastic();
```
Comment (shadyvb) [inline tests/acceptance/ElasticSearchCest.php:298]:
```suggestion
		$I->reindexContent();
```

**Review by roborourke — APPROVED** [review]

**Comment by shadyvb** [comment]
Suspecting this one might be failing because of the introduction of the new default theme 🤔 will need to deal with that separately. The `reindexContent` function invokation has been tested on `altis/cms` and seems to be working as expected.

**Comment by roborourke** [comment]
I thought the same but running locally with the Altis test theme worked. Maybe something got mixed up still though, we could try a PR that removes the custom stuff from this module's `.travis.yml`

**Comment by shadyvb** [comment]
Yes, that was my next step as well.

**Pull request merged**